scripts/release-api.sh

```diff
@@ -21,8 +21,8 @@ if [[ -z ${RELEASE_API_VERSION} ]]; then
 fi
 
 echo "Setting API version to ${RELEASE_API_VERSION}"
-mvn -q versions:set -DnewVersion="${RELEASE_API_VERSION}" -DprocessAllModules=true -pl ${API_MODULES} -DgenerateBackupPoms=false
-mvn -q clean install -Pquick -pl ${API_MODULES}  #quick sanity check to ensure the API modules still build
+mvn -q versions:set -DnewVersion="${RELEASE_API_VERSION}" -DprocessAllModules=true -pl :kroxylicious-bom,{API_MODULES} -DgenerateBackupPoms=false
```

Comment:
missing a $ there? `:kroxylicious-bom,${API_MODULES}`

Reply:
fixed in https://github.com/kroxylicious/kroxylicious/pull/506/commits/300126e9816b33adab492305f977e354c8a4c989